src/Jackett.Common/Definitions/newstudio.yml

```diff
@@ -0,0 +1,103 @@
+﻿---
+  site: newstudio
+  name: Newstudio
```

Comment:
Please add NewStudio to the readme.md in the 'Supported Semi-Private Trackers' section

Reply:
done